eel_hole/logs.py

```diff
@@ -0,0 +1,32 @@
+"Centralized logging configuration."
+
+import logging
+
+import structlog
+from flask_login import current_user
+
+
+def user_id_adder(logger, log_method, event_dict):
+    """Add user ID to log if available.
+
+    Note that there is a difference between `user_id` not existing (no user in
+    context) and `user_id: null` (anonymous user).
```

Comment:
Checking my understanding:

* When there is no user in context, line 19 will not execute, and "user_id" will not be added to the structlog
* When there is a user in context but they are not logged in, line 19 will execute, and "user_id" will be added to the structlog with value "null"
* When there is a user in context and they are logged in, line 19 will execute, and "user_id" will be added to the structlog with the integer id for that logged-in user

if so then I want to adjust this note to explicitly be about what gets added to the log event (or not), so it doesn't get mixed up with the assumptions made by this function

```suggestion
    When there is no user in context (such as during app startup), `user_id`
    will not be added to the log event.
    When there is a user in context but the user is anonymous, `user_id` will be
    added to the log event, but with value `null`.
```

would also go for "included in" instead of "added to" if that makes more sense for ETL readers